addons/mail/static/tests/crosstab/crosstab.test.js

```diff
@@ -179,13 +179,13 @@ test("Message (hard) delete notification", async () => {
     await start();
     await openDiscuss("mail.box_inbox");
     await click("[title='Add Star']");
-    await contains("button", { text: "Inbox", contains: [".badge", { text: "1" }] });
-    await contains("button", { text: "Starred messages", contains: [".badge", { text: "1" }] });
+    await contains("button:has(:text('Inbox'))", { contains: [".badge:text('1')"] });
+    await contains("button:has(:text('Starred messages'))", { contains: [".badge:text('1')"] });
     const [partner] = pyEnv["res.partner"].read(serverState.partnerId);
     pyEnv["bus.bus"]._sendone(partner, "mail.message/delete", {
         message_ids: [messageId],
     });
     await contains(".o-mail-Message", { count: 0 });
-    await contains("button", { text: "Inbox", contains: [".badge", { count: 0 }] });
-    await contains("button", { text: "Starred messages", contains: [".badge", { count: 0 }] });
+    await contains("button:has(:text('Inbox'))", { contains: [".badge", { count: 0 }] });
+    await contains("button:has(:text('Starred messages'))", { contains: [".badge", { count: 0 }] });
```

Comment:
The innner `contains` could be replaced with `:has()` too, but ok to make that in follow-up PR that removes the internal `contains:`.